web/src/helpers/redux.js

```diff
@@ -63,7 +63,8 @@ function initDataState() {
   });
 
   return {
-    hasConnectionWarning: false,
+    failedRequestType: null,
+    failedRequestZoneId: null,
```

Comment:
I'm wondering whether it is necessary to have the specific failedRequestZoneId stored in state. In other parts of the application we are retrieving the current zoneId through

const { zoneId } = useParams();

Since the user is only able to see one zone at any given time, the failedRequestZoneId should also always be the same as the one in params.